spyder/api/plugins.py

```diff
@@ -472,6 +472,21 @@ def update_font(self):
         """
         pass
 
+    def update_style(self):
```

Comment:
@ccordoba12 you agree with this name for updating icons and styles?

Reply:
Sure, that's fine.